libs/ubpf/kernel/ubpf_kernel.c

```diff
@@ -89,6 +89,37 @@ ubpf_jit_update_helper_x86_64(
     return false;
 }
 
+// Thunk out JIT related calls.
+// Workaround until https://github.com/iovisor/ubpf/issues/185 is fixed.
+struct ubpf_jit_result
+ubpf_translate_arm64(struct ubpf_vm* vm, uint8_t* buffer, size_t* size, enum JitMode jit_mode)
+{
+    __fastfail(0);
```

Comment:
Does _failfast report logging. Otherwise, please consider adding logging or other error handling mechanisms to provide more context when failure happen. This comment applies to the below functions too!

Reply:
This will generate a bugcheck, so there will be a kernel dump and / or a watson report.